v8py/convert.cpp

```diff
@@ -38,7 +39,8 @@ PyObject *py_from_js(Local<Value> value, Local<Context> context) {
     if (value->IsArray()) {
         Local<Array> array = value.As<Array>();
         PyObject *list = PyList_New(array->Length());
-        for (unsigned i = 0; i < array->Length(); i++) {
+        uint32_t length = array->Length();
+        for (uint32_t i = 0; i < length; i++) {
```

Comment:
This seems a little pointless, unless it fixes some compiler error? Also see line 63 and 262 of this file